app/controllers/application_controller.rb

```diff
@@ -173,4 +182,8 @@ def sp_session
   def render_not_found
     render template: 'pages/page_not_found', layout: false, status: 404, formats: :html
   end
+
+  def render_timeout
+    render template: 'pages/page_took_too_long', layout: false, status: 503, formats: :html
```

Comment:
Thoughts on naming this page `503.html` to stay consistent with our other pages that map to HTTP status codes?

Reply:
Thoughts on adding an analytics event so we can track what kind of timeouts we get?

Reply:
That might work. I was going with the `pages/page_not_found` that's used for the 404 as a pattern.

Reply:
Added analytics. Not sure if that's the right data/keys we want, but those are easy to change.